runtime/shared_common/CompiledMethodManagerImpl.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2001, 2019 IBM Corp. and others
+ * Copyright (c) 2001, 2020 IBM Corp. and others
```

Comment:
This change is unnecessary.

Reply:
It's the only change in this file, so the copyright doesn't need to be updated.

Reply:
Probably I touched it somehow, but then reverted the change. In any case I can revert it.

Reply:
Reverted.